tools/trace_bind.txt

```diff
@@ -0,0 +1,100 @@
+Demonstrations of trace_bind, the Linux eBPF/bcc version.
```

Comment:
This file's name should be trace_bind_example.txt.

Reply:
Renaming to bindsnoop_example.txt